docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md

```diff
@@ -858,24 +858,24 @@ If you need to reconfigure a node agent, you can use the following procedure:
     ```
 
     ```output
-    * The current value of Node Name is set to node1; Enter new value or enter to skip: 
-    * The current value of Node IP is set to 10.9.82.61; Enter new value or enter to skip: 
+    * The current value of Node Name is set to node1; Enter new value or enter to skip:
+    * The current value of Node IP is set to 10.9.82.61; Enter new value or enter to skip:
     * Select your Onprem Provider.
```

Comment:
This has changed to 
**Select your Onprem Manually Provisioned Provider**

in 2.18.6 or 2.20.2 or later to indicate that it applies only to manually provisioned providers.

Reply:
okay ill change this for the 2.18 and 2.20 pages.